heareval/tasks/pipeline.py

```diff
@@ -458,15 +461,20 @@ def run(self):
                 f"{num_files} audio files in corpus."
                 f"Max files to subsample: {max_files}"
```

Comment:
Can we update this to include the split name? "{num_files} audio files in {self.split}. Max files ..."

This is different thing but it would be great to update all the print statements to use the luigi logger. logger.info("")

Reply:
Done.

Have you used the luigi logger?

Reply:
Yep, we use it in a couple places. It's pretty easy:

```
import logging
logger = logging.getLogger("luigi-interface")
logger.info("some message")
```

With that caveat that logger.info needs to be called within the pipeline. For example, we are using it outside of the pipeline in runner.py in a couple places, and those messages are not being logged - so we should actually fix that.